Framework/DataHandling/src/SaveAscii2.cpp

```diff
@@ -443,11 +443,11 @@ void SaveAscii2::populateSpectrumNumberMetaData() {
 void SaveAscii2::populateAngleMetaData() {
   std::vector<std::string> angles;
   const size_t nHist = m_ws->getNumberHistograms();
+  const auto &spectrumInfo = m_ws->spectrumInfo();
   for (size_t i = 0; i < nHist; i++) {
     const auto specNo = m_ws->getSpectrum(i).getSpectrumNo();
     const auto workspaceIndex = m_specToIndexMap[specNo];
```

Comment:
Am I missing something here? Is there something special in `m_specToIndexMap` or are this two lines translating workspace-index -> spectrum-number -> workspace-index, i.e., do nothing?